javascript/lib/dependabot/bun.rb

```diff
@@ -0,0 +1,36 @@
+# typed: strict
+# frozen_string_literal: true
+
+require "zeitwerk"
+
+loader = Zeitwerk::Loader.new
+
+# Set autoload paths for common/lib, excluding files whose content does not match the filename
+loader.push_dir(File.join(__dir__, "../../../common/lib"))
+loader.ignore(File.join(__dir__, "../../../common/lib/dependabot/errors.rb")) # we need to require these
+loader.ignore(File.join(__dir__, "../../../common/lib/dependabot/logger.rb")) # we need to require these
+loader.ignore(File.join(__dir__, "../../../common/lib/dependabot/notices.rb")) # we need to require these
+loader.ignore(File.join(__dir__, "../../../common/lib/dependabot/clients/codecommit.rb")) # we need to require these
```

Comment:
The classes in these files do not match the file names/paths. We can manually require these in this file with a comment until they can be resolved later.